std/random.d

```diff
@@ -52,7 +52,7 @@ License:   $(HTTP www.boost.org/LICENSE_1_0.txt, Boost License 1.0).
 Authors:   $(HTTP erdani.org, Andrei Alexandrescu)
            Masahiro Nakagawa (Xorshift random generator)
            $(HTTP braingam.es, Joseph Rushton Wakeling) (Algorithm D for random sampling)
-           Ilya Yaroshenko (Mersenne Twister implementation, adapted from $(HTTPS github.com/libmir/mir-random, mir.random))
+           Ilya Yaroshenko (Mersenne Twister implementation, adapted from $(HTTPS github.com/libmir/mir-random, Mir Random))
```

Comment:
It's still showing a broken link. I think you need to prepend `random` with an underscore, like `mir._random`.

Yeah.  This is proof that this autohighlighting misfeature is really, really dumb and needs to be killed with fire.